middleware/index.js

```diff
@@ -0,0 +1,29 @@
+const bcrypt = require("bcryptjs");
+
+const hash = (req, res, next) => {
+  let user = req.body;
+  if (user.password.length < 8) {
+    res.status(400).json({ message: "Password length must be 8 characters" });
+  }
+  if (!user.password) {
+    res.status(400).json({ message: "You must enter a password" });
+  } else {
+    const hashPass = bcrypt.hashSync(user.password, 10);
```

Comment:
Should we increase the second parameter of the hashSync function? Just starting a conversation about it --  we should revisit this when we release to production :)